app/middleware/logging.py

```diff
@@ -17,7 +17,12 @@ def __call__(self, request):
         # Code to be executed for each request before
         # the view (and later middleware) are called.
         logger.info(
-            "Request %s %s", request.method.upper(), request.path, extra={"request": request}
+            "Request %s %s",
+            request.method.upper(),
+            request.path,
+            extra={
+                "request": request, "requestPayload": str(request.body[:200])
```

Comment:
Same here we need to log the payload only for POST, PUT, PATCH request

Reply:
Done